README.md

```diff
@@ -10,3 +10,5 @@ Civil Society Registry Sofware
 - `php artisan migrate`
 - `php artisan orchid:admin admin admin@admin.com password` [Create a user with maximum permissions with name: admin, email: admin@admin.com and password: password. Change accordingly]
 - `php artisan serve`
+- `./vendor/bin/pint` [Format the codebase]
```

Comment:
This is not the best way to do it.

Please, add a new section called #Contributing in that section you can add a sub-section on on ## Linting and describe how linting is done and mention that contributors need to make sure their PRs pass the lint check before asking for reviews.

Hope that makes sense.